src/include/network/lan/bridge.rb

```diff
@@ -34,16 +34,43 @@ def initialize_network_lan_bridge(_include_target)
       textdomain "network"
     end
 
+    # It returns an empty string if given IP is 0.0.0.0, otherwise returns
+    # the given IP
+    #
+    # @param [String] ip
+    # @return [String] empty string if given ip is 0.0.0.0; given ip
+    # otherwise
+    def bridge_ip(ip)
+      ip == "0.0.0.0" ? "" : ip.to_s
+    end
+
+    # Checks if the given interface is using the old way of config enslaved
+    # interfaces with bootproto as static and 0.0.0.0 as IPADDR
+    #
+    # @param [String] interface name
+    # @return [Boolean] returns true if given enslaved interface is configured
+    # in the old way
+    def old_bridge_port_config?(ifcfg_name)
+      current = LanItems.GetDeviceMap(LanItems.find_configured(ifcfg_name))
+      return false unless current
+
+      current["BOOTPROTO"] == "static" && current["IPADDR"] == "0.0.0.0"
+    end
+
     # Immediately updates device's ifcfg to be usable as bridge port.
     #
     # It mainly setups suitable BOOTPROTO an IP related values
     def configure_as_bridge_port(device)
+      selected_interface = NetworkInterfaces.Current
       log.info("Adapt device #{device} as bridge port")
 
       # when using wicked every device which can be bridged
       # can be set to BOOTPROTO=none. No workaround with
       # BOOTPROTO=static required anymore
-      NetworkInterfaces.Edit(device)
+      unless NetworkInterfaces.Edit(device)
+        Lan.autoconf_slaves += [device] unless Lan.autoconf_slaves.include? device
+        return false
```

Comment:
@teclator what is this method supposed to return in this case, and in the normal case? Add a @return tag?